troi/tools/apple_lookup.py

```diff
@@ -1,7 +1,13 @@
 import requests
+from ratelimit import limits, sleep_and_retry
 
 APPLE_MUSIC_URL = f"https://api.music.apple.com/"
+CALLS=5
```

Comment:
CALLS isn't a good name -- I can't quite deduce what it means from its name. Also, I really like to note units in the commends; what is the unit of RATE_LIMIT?

Reply:
removed altogether now.